screen_widgets.go

```diff
@@ -1,287 +1,201 @@
+	Color      *string `json:"color,omitempty"`
+	Palette    *string `json:"palette,omitempty"`
+	Comparator *string `json:"comparator,omitempty"`
+	Invert     *bool   `json:"invert,omitempty"`
+	Value      *string `json:"value,omitempty"`
+	ImageURL   *string `json:"image_url,omitempty"`
 }
 
-type AlertGraphWidget struct {
-	TitleSize    *int    `json:"title_size,omitempty"`
-	VizType      *string `json:"timeseries,omitempty"`
-	Title        *bool   `json:"title,omitempty"`
-	TitleAlign   *string `json:"title_align,omitempty"`
-	TitleText    *string `json:"title_text,omitempty"`
-	Height       *int    `json:"height,omitempty"`
-	Width        *int    `json:"width,omitempty"`
-	X            *int    `json:"y,omitempty"`
-	Y            *int    `json:"x,omitempty"`
-	AlertId      *int    `json:"alert_id,omitempty"`
-	Timeframe    *string `json:"timeframe,omitempty"`
-	Type         *string `json:"type,omitempty"`
-	AddTimeframe *bool   `json:"add_timeframe,omitempty"`
+type TileDefRequestStyle struct {
+	Palette *string `json:"palette,omitempty"`
+	Type    *string `json:"type,omitempty"`
+	Width   *string `json:"width,omitempty"`
 }
 
-type HostMapWidget struct {
-	TitleSize  *int     `json:"title_size,omitempty"`
-	Title      *bool    `json:"title,omitempty"`
-	TitleAlign *string  `json:"title_align,omitempty"`
-	TitleText  *string  `json:"title_text,omitempty"`
-	Height     *int     `json:"height,omitempty"`
-	Width      *int     `json:"width,omitempty"`
-	X          *int     `json:"y,omitempty"`
-	Y          *int     `json:"x,omitempty"`
-	Query      *string  `json:"query,omitempty"`
-	Timeframe  *string  `json:"timeframe,omitempty"`
-	LegendSize *int     `json:"legend_size,omitempty"`
-	Type       *string  `json:"type,omitempty"`
-	Legend     *bool    `json:"legend,omitempty"`
-	TileDef    *TileDef `json:"tile_def,omitempty"`
+type TileDefStyle struct {
+	Palette     *string `json:"palette,omitempty"`
+	PaletteFlip *string `json:"paletteFlip,omitempty"`
+	FillMin     *string `json:"fillMin,omitempty"`
+	FillMax     *string `json:"fillMax,omitempty"`
 }
 
-type CheckStatusWidget struct {
-	TitleSize  *int    `json:"title_size,omitempty"`
-	Title      *bool   `json:"title,omitempty"`
-	TitleAlign *string `json:"title_align,omitempty"`
-	TextAlign  *string `json:"text_align,omitempty"`
-	TitleText  *string `json:"title_text,omitempty"`
-	Height     *int    `json:"height,omitempty"`
-	Width      *int    `json:"width,omitempty"`
-	X          *int    `json:"y,omitempty"`
-	Y          *int    `json:"x,omitempty"`
-	Tags       *string `json:"tags,omitempty"`
-	Timeframe  *string `json:"timeframe,omitempty"`
-	TextSize   *string `json:"text_size,omitempty"`
-	Type       *string `json:"type,omitempty"`
-	Check      *string `json:"check,omitempty"`
-	Group      *string `json:"group,omitempty"`
-	Grouping   *string `json:"grouping,omitempty"`
+type Time struct {
+	LiveSpan *string `mapstructure:"live_span" json:"live_span,omitempty"`
```

Comment:
There's no reference to the mapstructure package in this repo. What's the intent of this tag? Will it be consumed downstream?

cc @yfronto

Reply:
My mistake, didn't add this on purpose. Leftover from #158. Nice catch!